Connection/Connection.php

```diff
@@ -110,6 +120,9 @@ public function testConnection(bool $throwExceptions = false): bool
 
     public function tryTestConnection(): void
     {
+        if(!$this->isEnabled()) {
+            throw new ConnectionException(array('Mailbox is not enabled'));
```

Comment:
I do not thin that we should thow here ConnectionException as it indicates that the problem is in PhpImap library, not in the bundle. I think we should thow here ErrorException. Additionally please do not use array() notation, please use [] instead.